src/public/modules/forms/base/directives/submit-form.directive.js

```diff
@@ -90,44 +88,29 @@ function submitFormDirective(
         $('head').append(meta)
       }
 
-      scope.formLogin = function (authType, rememberMe) {
-        if (authType === 'MyInfo') {
-          return $q
-            .when(MyInfoService.createRedirectURL(scope.form._id))
-            .then((response) => {
-              setReferrerToNull()
-              $window.location.href = response.redirectURL
-            })
-            .catch((error) => {
-              console.error(error)
-              Toastr.error(
-                'Sorry, there was a problem with your login. Please try again.',
-              )
-            })
-        }
+      scope.formLogin = function (authType, isPersistentLogin) {
```

Comment:
you don't even need the `authType` parameter here right?

Reply:
good catch! thanks for reminding me and i'll remove it